packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js

```diff
@@ -10,33 +10,49 @@ import ProjectStatistics from '@shared/components/ProjectStatistics'
 import ZooniverseTalk from './components/ZooniverseTalk'
 import ThemeModeToggle from '@components/ThemeModeToggle'
 import ProjectAnnouncement from './components/ProjectAnnouncement'
+import { Media } from '../../shared/components/Media'
```

Comment:
Non-blocking: I have to second that the naming of this is confusing. We should rename this one to be reflective of it being for responsive layout.